PalasoUIWindowsForms/ClearShare/LicenseInfo.cs

```diff
@@ -73,9 +76,46 @@ public virtual bool EditingAllowed
 
 	public class NullLicense : LicenseInfo
 	{
-		public override string GetDescription(string iso639_3LanguageCode)
+		/// <summary>
+		/// Get a simple, non-legal summary of the license, using the "best" language for which we can find a translation.
+		/// </summary>
+		/// <param name="iso6393LanguageCode">A single language to try and use for the description.</param>
+		/// <returns>The description of the license if we have it in this language, else the English</returns>
+		public override string GetDescription(string iso6393LanguageCode)
+		{
+			string idOfLanguageUsed;
+			return GetDescription(new string[] {iso6393LanguageCode, "en"}, out idOfLanguageUsed);
+		}
+
+		/// <summary>
+		/// Get a simple, non-legal summary of the license, using the "best" language for which we can find a translation.
+		/// </summary>
+		/// <param name="languagePriorityIds"></param>
+		/// <param name="idOfLanguageUsed">The id of the language we were able to use. Unreliable if we had to use a mix of languages.</param>
+		/// <returns>The description of the license.</returns>
+		public override string GetDescription(IEnumerable<string> languagePriorityIds, out string idOfLanguageUsed)
```

Comment:
Looks very like GetComponentOfLicenseInBestLanguage. Can we abstract some of this logic into a base class method?